### PR TITLE
[KIECLOUD-507] - Console-form-cr checkbox doesn't work with default v…

### PR DESCRIPTION
**Pull request by swati-kale**
…alue

Signed-off-by: Swati Kale <swkale@redhat.com>

1)Fixed - UI was not loading on the latest code giving appendTo=null error
2)Added default check for checkbox field and handle the same in yaml generation
3)Added the same in sample full-form.json

### CODE REVIEW

File: frontend/src/component/operator-wizard/OperatorWizard.js [hunk 1]
```diff
@@ -319,51 +319,55 @@ export default class OperatorWizard extends Component {
...
-              let subPageFields = subPage.fields;
+            if (subPage.visible !== undefined && subPage.visible === true) {
+              if (
+                (subPage.label === SMART_ROUTER_STEP ||
+                  subPage.label === PIM_STEP) &&
+                this.stepBuilder
+                  .getObjectMap(ENV_KEY)
+                  .startsWith(RHDM_ENV_PREFIX)
+              ) {
+                //do not add in yaml
```
Comment (yzhao583) [inline frontend/src/component/operator-wizard/OperatorWizard.js:330]: Is it just a empty {} here?

File: frontend/src/component/operator-wizard/OperatorWizard.js [hunk 2]
```diff
@@ -513,8 +519,10 @@ export default class OperatorWizard extends Component {
         </Modal>
 
         <Modal
+          id="errorMsgModal"
           isSmall
           title="Review the form"
+          appendTo={() => document.body}
```
Comment (yzhao583) [inline frontend/src/component/operator-wizard/OperatorWizard.js:525]: Ah, here we go :)

File: frontend/src/component/operator-wizard/OperatorWizard.js (hunk 1 above)
Comment (swati-kale) [reply frontend/src/component/operator-wizard/OperatorWizard.js:330]: It is existing code, was added earlier to skip yaml generation for smartrouter and pim for RHDM environments.

**Comment by yzhao583** [comment]
I think it is good to go for this quick fix

**Comment by swati-kale** [comment]
> I think it is good to go for this quick fix

@bmozaffa Can it be merged?

**Pull request merged**